Poker.BE/Poker.BE.DomainTests/Security/UserManagerTests.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Poker.BE.Domain.Security;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Poker.BE.Domain.Security.Tests
+{
+    [TestClass()]
+    public class UserManagerTests
+    {
+        [TestMethod()]
+        public void UserManagerTest()
+        {
+            // simple constructor
```

Comment:
if that a simple constructor - delete the test.
when you leave it like that it just pass without any reason. 👎